### ISSUE TITLE
format of numbers - open for discussion

### ISSUE THREAD

**Issue by riwa73** [nicoeinsidler/pigor#8]
I dont like the long numbers .
maybe stick to only 2, 3 or 4 decimal places.
May even depend on the integer part

**Comment by nicoeinsidler** [comment]
Which numbers do you mean? In the created analysis files (md and html)? This is easy fixable, but we should think about a case where it is more useful to see a rounded solution. 

Stephan, Armin and the others should comment on that as well.

**Comment by riwa73** [comment]
Yes - maybe have only a truncated/nicely formatted output in the html and the full accuracy in the md-output.
But I'm with you to discuss it with the rest of the group
